plugins/single-instance/examples/vanilla/src-tauri/src/main.rs

```diff
@@ -9,6 +9,7 @@
 
 fn main() {
   tauri::Builder::default()
+    .plugin(tauri_plugin_cli::init())
```

Comment:
I believe we'll likely want to remove this last commit which modified some files in the example. Including the CLI was the main (only?) way to test this on MacOS and I wanted to make it easy for you folks to try it out yourself. I can remove it when you are ready for it, let me know about how you'd like to tackle this.

Reply:
as far as i know this is not required to start the app from the console. If it is then imo it's fine to keep the cli plugin in the example.